pkg/storaged/stratis/pool.jsx

```diff
@@ -464,69 +653,244 @@ const StratisPoolCard = ({ card, pool, degraded_ops, can_grow, stats }) => {
+                                        <StorageLink
+                                              onClick={() => remove_tang(clevis_infos[0])}
+                                              excuse={remove_tang_excuse}
+                                        >
+                                            {_("Remove")}
+                                        </StorageLink>
+                                    </FlexItem>
+                                </>
+                                : <FlexItem>
+                                    {cockpit.format(_("Clevis \"$0\""), clevis_infos[0].pin)}
+                                </FlexItem>
+                            )
+                        }
+                    </Flex>
+                </StorageDescription>
+            </>
+        );
+    } else {
+        let add_excuse;
+        if (key_descs.length + clevis_infos.length >= 15)
+            add_excuse = _("No more space for passphrases or keyservers.");
+
+        const KeyDescSlot = ({ info }) => (
+            <Tr data-test-row-location="passphrase">
+                <Td>{cockpit.format(_("Slot $0"), info.slot)}</Td>
+                <Td>
+                    {_("Passphrase")}
+                </Td>
+                <Td modifier="nowrap" className="pf-v6-c-table__action">
+                    <Actions onlyMenu actions={[
+                        {
+                            title: _("Change"),
+                            action: () => change_passphrase(info),
+                        },
+                        {
+                            title: _("Remove"),
+                            action: () => remove_passphrase(info),
+                            excuse: remove_passphrase_excuse,
+                            danger: true,
+                        },
+                    ]} />
+                </Td>
+            </Tr>
+        );
+
+        const TangUrlSlot = ({ info }) => (
+            <Tr data-test-row-location={info.url}>
+                <Td>{cockpit.format(_("Slot $0"), info.slot)}</Td>
+                <Td>
+                    {info.url}
+                </Td>
+                <Td className="pf-v6-c-table__action">
+                    <Actions onlyMenu actions={[
+                        {
+                            title: _("Remove"),
+                            action: () => remove_tang(info),
+                            excuse: remove_tang_excuse,
+                            danger: true,
+                        },
+                    ]} />
+                </Td>
+            </Tr>
+        );
+
+        const OtherClevisSlot = ({ info }) => (
+            <Tr data-test-row-location={info.pin}>
+                <Td>{cockpit.format(_("Slot $0"), info.slot)}</Td>
+                <Td>{cockpit.format(_("Clevis \"$0\""), info.pin)}</Td>
+            </Tr>
+        );
+
+        // eslint-disable-next-line no-inner-declarations
+        function make_row(tok) {
```

Comment:
Move it outside if-statement still. Wouldn't change much but makes the function declaration a bit better. Or split this pool creation into separate components